src/app/CMakeLists.txt

```
@@ -1,4 +1,6 @@
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR}
+    COMPONENTS Concurrent Network WebChannel WebEngineWidgets Widgets
+    REQUIRED)
```

As @intelfx pointed out, it's enough to add `Concurrent` and `WebEngineWidgets`. Just tested and that works.

```suggestion
find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Concurrent WebEngineWidgets Widgets REQUIRED)
```